src/mjlab/envs/mdp/events.py

```diff
@@ -517,7 +518,10 @@ def randomize_pd_gains(
   if isinstance(asset_cfg.actuator_ids, list):
     actuators = [asset.actuators[i] for i in asset_cfg.actuator_ids]
   else:
-    actuators = asset.actuators[asset_cfg.actuator_ids]
+    actuators = [asset.actuators[asset_cfg.actuator_ids]]
+
+  # Unwrap DelayedActuators to access base actuators.
+  actuators = [a._base_actuator if isinstance(a, DelayedActuator) else a for a in actuators]
```

Comment:
Can you add base_actuator as a property and use it here instead of accessing the private attribute?